kge/dataset.py

```diff
@@ -91,6 +93,53 @@ def load(config: Config, preload_data=True):
                 dataset.split(split)
         return dataset
 
+    @staticmethod
+    def create_from(
+        checkpoint: Dict,
+        config: Config,
+        dataset: Optional[Dataset] = None,
+        preload_data=True,
```

Comment:
Perhaps better to use preload=False as default. This makes it easier to use packages. I am actually not sure if we should support preloading at all.